link/src/test/java/com/stripe/android/link/account/LinkAccountManagerTest.kt

```diff
@@ -183,17 +183,31 @@ class LinkAccountManagerTest {
             val accountManager = accountManager()
             val phone = "phone"
             val country = "country"
+            val name = listOf("name", null).random()
```

Comment:
This is meant to show that we don’t care about the actually sent value, but just want it to be the same when `linkRepository.consumerSignUp()` is called.